Extensions/NotificationService/NotificationService.swift

```diff
@@ -82,15 +82,15 @@ class NotificationService: UNNotificationServiceExtension {
 class SyncDataDisplay {
     var contentHandler: ((UNNotificationContent) -> Void)
     var notificationContent: UNMutableNotificationContent
-    var sentTabs: [SentTab]
+    // var sentTabs: [SentTab]
```

Comment:
This appeared to be unused. Do we do anything with `sentTabs` in the notification, @garvankeeley?

Reply:
agree it looks unused